### ISSUE TITLE
Tow Job- Repo

### ISSUE THREAD

**Issue by M4TT3R1** [InZidiuZ/op-framework-issue-tracker#2765]
### Assurance

I have read the `README.md`

### Summary

Suggestion to add a Repo job for mechanics/tow drivers. 

Similar to a chop radio, but is legal to own. Every so often (30min - hour) a request would come in for specific vehicle. It would generate an address similar to chop.  Tow driver would tow car and drop off at the southside scrap yard near Maze Arena. Would pay small amount ($50-100).

Each Repo would have a chance to claim 1-3 of a material(scrap, aluminum, steel, rubber, glass) and also a second opportunity to obtain a fake license plate, basic repair kit, first aid kit, or food.

### Reason

There are 3 ways for mechanics to obtain materials and only one other way to obtain fake plates. This would open up mech shops to hire tow drivers and give them a purpose instead of waiting for the tow line only. It gives an alternate method of obtaining mats legally instead of scuba and it opens up more fake plates in the city. 

Currently, barely any crims even try to use fake plates in any heists. Making some more readily available opens up more interactions by having mechs sell fake plates to civilians, and hopefully, encourages crims to put in more effort in their heists which would give PD more work to do instead of instantly knowing who is doing the bank before the heist even happens.

**Comment by ImArteria** [comment]
Gonna be honest the entire thought process of having more fake plates means people will sell more is the opposite of what will happen. 

More fake plates means theyll be even less valuable than they are. Low Demand high Supply means low price. 

Overall the idea itself is cool theoretically if people would actually tow and not prefer a mechanic.

**Comment by M4TT3R1** [comment]
i think most crims are too lazy to go do the work to get a fake plate so they just put no effort into the crim. this would justt make it a little easier and hopefully encourage more RP. i dont look at it as a money making opportunity selling the plates jsut something to hopefully spike some decent crim rp

**Comment by enutpeanut** [comment]
> Gonna be honest the entire thought process of having more fake plates means people will sell more is the opposite of what will happen.
Personally would rather fake plates more accessible, because it means we don't keep seeing the same car outside a bank 15 times a day and the criminals are putting in no effort into covering up their identities. I would prefer fake plates to be more common than they currently are, so the criminal roleplay is slightly improved.

**Comment by vol5** [comment]
The idea overall is dope.I like it more if the shops have to compete similar to chop. Loot pool would have to be slightly different because even though frequency is much lower it can't have the same rewards as chop or better since it's a legal gig..

**Comment by coalaura** [comment]
This is literally an exact replica of chop, why would be add the same thing twice??? Doesn't really make sense to me. If all you want is to get more fake plates then we could look into upping the drop chances in chop or other places if it indeed is an issue which i do not think it is.

Also from a roleplay standpoint, towing a random car and scrapping it is literally chop, what would make this different and therefore legal? I'm struggling to see the use of this.

This seems a bit like you just want to legalize chop to get materials quicker. Materials are currently perfectly balanced. It should require effort to get them and if you want to get more materials you can do chop which will get you materials quicker while having the added risk of it being illegal.

TL;DR: No

**Comment by coalaura** [comment]
Adding to this, you can very easily do all of this in roleplay. You can roleplay having gotten a call to tow a local vehicle if PD stops you when doing chop
